webdriver/tests/element_click/element_click.py

```diff
@@ -0,0 +1,81 @@
+import pytest
+
+from tests.support.asserts import assert_error, assert_success
+from tests.support.inline import inline
+
+def click(session, element):
+    return session.transport.send("POST", "session/{session_id}/element/{element_id}/click"
+                                  .format(session_id=session.session_id,
+                                          element_id=element.id))
```

Comment:
I made a huge refactoring of the tests on https://bugzilla.mozilla.org/show_bug.cgi?id=1455568 which is going to be synced upstream soon (hopefully tomorrow). Mind aligning the style/indentation of this code to what is used there?

Reply:
fixed